libraries/AP_GyroFFT/CMSIS_5/include/arm_math.h

```diff
@@ -346,7 +346,7 @@ extern "C"
 #define DELTA_Q15          0x5
 #define INDEX_MASK         0x0000003F
 #ifndef PI
-  #define PI               3.14159265358979f
+  #define PI               (M_PI)
```

Comment:
Should we really be modifying `CMSIS_5` source code?  @andyp1per

Reply:
Yes let's not modify it

Reply:
Thanks @andyp1per  - we really ought not leave these little landmines around for other people to find, but I can't see how we can sensibly put warnings on these things for people to find.